app/controllers/application_controller.rb

```diff
@@ -4,4 +4,6 @@ class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
   before_action :authenticate_user!
+
+  add_flash_types :alert_with_description
```

Comment:
I've not seen this before, is this so you don't need to put things in a `flash:` hash?

Reply:
Yep! Some other apps [use this too](https://github.com/search?q=org%3Aalphagov+add_flash_types&type=Code).